src/SSVOpenHexagon/Global/Config.cpp

```diff
@@ -162,7 +165,22 @@ static void syncAllToObj()
 #undef X
 }
 
+static void resetAllFromObj()
+{
+#define X(name, type, key) name().syncFrom(ssvuj::getFromFile("../misc/default_config.json"));
```

Comment:
In the deployed game, `default_config.json` is in the main folder, not under `../misc`. I think this needs to be changed to `./default_config.json` (and below as well). Can you test this by copy-pasting your build in your Steam Open Hexagon folder (and replacing the exe)?

Reply:
I tried using default_config.json in the github build but it would always crash.
I am going to check what happens with the Steam game.